File: Sources/SimERP.Data/DBEntities/Customer.cs
using System;
using System.Collections.Generic;

namespace SimERP.Data.DBEntities
{
    public partial class Customer
    {
        public long CustomerId { get; set; }
        public string CustomerCode { get; set; }
        public string CustomerName { get; set; }
        public string Address { get; set; }
        public string PhoneNumber { get; set; }
        public string FaxNumber { get; set; }
        public string Email { get; set; }
        public int? GroupCompanyId { get; set; }
        public int? CustomerTypeId { get; set; }
        public string CustomerTypeList { get; set; }
        public int? PaymentTermId { get; set; }
        public decimal DebtCeiling { get; set; }
        public string TaxNumber { get; set; }
        public string CompanyName { get; set; }
        public string CompanyAddress { get; set; }
        public string BankingNumber { get; set; }
        public string BankingName { get; set; }
        public string Notes { get; set; }
        public string RepresentativeName { get; set; }
        public string RepresentativePhone { get; set; }
        public string RepresentativeAddress { get; set; }
        public string RepresentativeEmail { get; set; }
        public int CreatedBy { get; set; }
        public DateTimeOffset CreatedDate { get; set; }
        public int? ModifyBy { get; set; }
        public DateTimeOffset? ModifyDate { get; set; }
        public string CountryId { get; set; }
        public int? ProvinceId { get; set; }
        public int? DistrictId { get; set; }
        public int? WardId { get; set; }
        public decimal? Latitude { get; set; }
        public decimal? Longitude { get; set; }
        public string SearchString { get; set; }
        public string TrackingNotes { get; set; }
        public bool IsGroupCompany { get; set; }
        public bool IsCompany { get; set; }
        public bool IsActive { get; set; }
        public int? SortOrder { get; set; }
    }
}

File: Sources/SimERP/ClientApp/src/app/lists/customer/model/customer.ts
import { Attachfile } from '../../product/model/attachfile';
import { CustomerProduct } from './customerproduct';
import { CustomerSale } from './customersale';
import { CustomerCommission } from './customercommission';
import { CustomerDelivery } from './customerdelivery';

export class Customer {
    CustomerId: any;
    CustomerCode: string = "";
    CustomerName: string = "";
    Address: string;
    PhoneNumber: string;
    FaxNumber: string;
    Email: string;
    GroupCompanyId: number;
    CustomerTypeId: number;
    CustomerTypeList: string;
    PaymentTermId: number;
    DebtCeiling: number;
    TaxNumber: string;
    CompanyName: string;
    CompanyAddress: string;
    BankingNumber: string;
    BankingName: string;
    Notes: string;
    RepresentativeName: string;
    RepresentativePhone: string;
    RepresentativeAddress: string;
    RepresentativeEmail: string;
    CreatedBy: number;
    CreateName: string = "";
    CreatedDate: Date;
    ModifyBy: number;
    ModifyDate: Date;
    CountryId: string;
    ProvinceId: number;
    DistrictId: number;
    WardId: number;
    Latitude: number;
    Longitude: number;
    SearchString: string;
    TrackingNotes: any;
    IsGroupCompany: boolean;
    IsCompany: boolean;
    IsActive: boolean = true;
    SortOrder: number;
    //tab hàng hóa
    objProduct: CustomerProduct[] = [];
    //tab saler
    objSaler: CustomerSale[] = [];
    //tab commission
    objCommission: CustomerCommission[] = [];
    //tab delivery
    objDelivery: CustomerDelivery[] = [];
    //tab file đính kèm
    ListAttachFile: Attachfile[] = [];
    ListAttachFileDelete: Attachfile[] = [];
}
